### PR TITLE
sys-cluster/kube-router Introducing new ebuild

### PR DESCRIPTION
**Pull request by podshumok**
Kube-router can be used instead of kube-proxy, and provides cni and ovelay network for Kubernetes it is a single daemon that can be configured to play all of those roles or just some of them.
Here is a ebuild written by myself

### CODE REVIEW

**Comment by podshumok** [comment]
> WARNING: The following maintainers do not match any Bugzilla accounts:

So I have a bugzilla account but with different email (and I prefer to keep it this way), is there a way to link few emails to one bugzilla account?

**Comment by mgorny** [comment]
> > WARNING: The following maintainers do not match any Bugzilla accounts:
> 
> So I have a bugzilla account but with different email (and I prefer to keep it this way), is there a way to link few emails to one bugzilla account?

Nope, Bugzilla does not support such a thing. However, metadata.xml is used only to assign bugs, so just put your Bugzilla address there.

**Comment by podshumok** [comment]
I think I'll just create a second bugzilla account :)

File: sys-cluster/kube-router/kube-router-0.3.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+KEYWORDS="~amd64"
+
+DESCRIPTION="Kube-router, a turnkey solution for Kubernetes networking"
+HOMEPAGE="https://kube-router.io"
+SRC_URI="${ARCHIVE_URI}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+
+DEPEND="dev-lang/go"
```
Comment (Zlogene) [inline sys-cluster/kube-router/kube-router-0.3.1.ebuild]: you do not need this dependency explicitly, eclasses will pull it for you.

File: sys-cluster/kube-router/kube-router-0.3.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+EAPI=7
+EGO_PN="github.com/cloudnativelabs/${PN}"
+
+inherit golang-build golang-vcs-snapshot
+
+ARCHIVE_URI="https://github.com/cloudnativelabs/kube-router/archive/v${PV}.tar.gz -> kube-router-${PV}.tar.gz"
+KEYWORDS="~amd64"
+
+DESCRIPTION="Kube-router, a turnkey solution for Kubernetes networking"
```
Comment (Zlogene) [inline sys-cluster/kube-router/kube-router-0.3.1.ebuild]: `A turnkey solution for Kubernetes networking` should be enough for `DESCRIPTION`

File: sys-cluster/kube-router/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>kpp+gentoo@qrator.net</email>
```
Comment (Zlogene) [inline sys-cluster/kube-router/metadata.xml]: you go first in metadata

File: sys-cluster/kube-router/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
```
Comment (Zlogene) [inline sys-cluster/kube-router/metadata.xml]: and proxy maintainers team goes last

**Comment by podshumok** [comment]
@Zlogene?

**Comment by podshumok** [comment]
and, @mgorny, I suppose?

**Pull request closed without merging**